### DISCUSSION TITLE
Suggestion for future iterations of OOTGW

### DISCUSSION THREAD

**Discussion by Jammers33** [oobabooga/text-generation-webui#4239] [Ideas]
It would be nice to see a "continue generating." prompt when the AI runs out of tokens, and it would be really awesome if that continuation happens in the same window as the first prompt generation, especially useful for research, or scripting responses that have more dynamic range. In addition, a copy function, check back, and scroll "up" and scroll "down" capabilities. oh, heck I am describing aspects of ChatGPT, yeah add those types of functions.    Also, it would be really groovy, if the UI had the abilities to have a user point to a recursive directory on desktop and upload directly for error analyzation whether it be a scripting language, or a thesis statement as a whole. Thank you for any considerations to this!

**Comment by TechScorpion** [comment]
> and scroll "up" and scroll "down" capabilities

There already are scroll up and down capabilities. You can scroll the past conversation by clicking and moving on the current position in the right side scroll bar of the past chat or moving the mouse wheel when your mouse pointer is over the conversation box.
